### ISSUE TITLE
Header of the stack has no background when title is defined

### ISSUE THREAD

**Issue by bieniu** [jcwillox/lovelace-canary#5]
When `vertical-stack` card has a `title` definied and the `in_stack: true` option is used, the header of the stack has no backgroud.

![image](https://user-images.githubusercontent.com/478555/83324761-ce053380-a267-11ea-85c8-b2772727a772.png)

Configuration:
```yaml
- type: vertical-stack
  title: "Kuchnia"
  in_card: true
  cards:
    - type: glance
      show_name: false
      entities:
        - sensor.temperature_kuchnia
        - sensor.humidity_kuchnia
    - type: entities
      show_header_toggle: false
      state_color: true
      entities:
        - binary_sensor.window_kuchnia
        - entity: sensor.dishwasher_kuchnia
```

**Comment by jcwillox** [comment]
Yes, it appears this is something I missed originally, it currently only encapsulates the child cards within the `ha-card` element, the title is left outside and so doesn't have a background.

I agree you would assume that the header is given a background as well, I think I assumed that if people wanted they could just add a title to the top card in the stack, in your case you could add the title to the glance card.

**Comment by jcwillox** [comment]
This has been fixed in the latest release [0.3.0](https://github.com/jcwillox/lovelace-canary/releases/tag/0.3.0) 👍
